ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Categories.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.ccd.sdk.api;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class Categories<R extends HasRole> {
```

Comment:
Shouldn't this be Category singular? And perhaps it should be CaseCategory?

Reply:
I have updated the class name to CaseCategory. Couldnt change the class to Singular. Getting an issue with Builder annotation. So kept is similar to .CaseRoleToAccessProfile class.